sdk/storage/Azure.Storage.Common/src/Shared/TransferValidationOptionsExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage
+{
+    internal static class TransferValidationOptionsExtensions
+    {
+        public static ValidationAlgorithm ResolveAuto(this ValidationAlgorithm validationAlgorithm)
```

Comment:
Do we need to worry about the null case?

Reply:
Can't be null. `ValidationAlgorithm` is an enum.